aws/vpc_flow_log_monitoring/lambda_function.py

```diff
@@ -51,16 +51,17 @@ def process_message(message, tags, timestamp, node_ip):
         detailed_tags.append("direction:outbound")
     if dstaddr == node_ip:
         detailed_tags.append("direction:inbound")
-
-    process_log_status(log_status, detailed_tags, timestamp)
-    if log_status == 'NODATA':
-        return
-
-    process_action(action, detailed_tags, timestamp)
-    process_duration(start, end, detailed_tags, timestamp)
-    process_packets(packets, detailed_tags, timestamp)
-    process_bytes(_bytes, detailed_tags, timestamp)
-
+    try:
+        process_log_status(log_status, detailed_tags, timestamp)
+        if log_status == 'NODATA':
+            return
+
+        process_action(action, detailed_tags, timestamp)
+        process_duration(start, end, detailed_tags, timestamp)
+        process_packets(packets, detailed_tags, timestamp)
+        process_bytes(_bytes, detailed_tags, timestamp)
+    except Exception, e:
+        print (e)
```

Comment:
Can we have add a friendly message, something like `"Could not process message: {}".format(e)`